docs/products/kafka/howto/enable-follower-fetching.md

```diff
@@ -22,21 +28,114 @@ physical location might have different AZ names in different accounts. To ensure
+To map AZ names to AZ IDs, see [AWS Knowledge Center article](https://repost.aws/knowledge-center/vpc-map-cross-account-availability-zones) and the [AWS documentation on AZ IDs](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html).
 
 ## Enable follower fetching
 
-Enable follower fetching with the [Aiven CLI](/docs/tools/cli):
+Use either of the following methods to enable follower fetching on your
+Aiven for Apache Kafka service:
+
+<Tabs groupId="config-methods">
+<TabItem value="cli" label="CLI" default>
+
+Enable follower fetching on an existing service with the Aiven CLI:
 
 ```bash
 avn service update <service-name> -c follower_fetching.enabled=true
 ```
 
 Parameters:
 
-- `<service-name>`: Replace with the actual name of your Kafka service.
+- `<service-name>`: Name of your Aiven for Apache Kafka service.
+- `follower_fetching={"enabled": true}`: Enables the follower fetching feature.
+
+</TabItem>
+<TabItem value="api" label="API">
+
+Use the [ServiceUpdate](https://api.aiven.io/doc/#tag/Service/operation/ServiceUpdate)
+API to enable follower fetching on an existing service:
+
+```bash
+curl --request PUT \
+    --url https://api.aiven.io/v1/project/YOUR_PROJECT_NAME/service/YOUR_SERVICE_NAME \
+    --header 'Authorization: Bearer YOUR_BEARER_TOKEN' \
+    --header 'content-type: application/json' \
+    --data '{
+        "user_config": {
+            "follower_fetching": {
+                "enabled": true
+            }
+        }
+    }'
+```
+
+Parameters:
+
+- `YOUR_PROJECT_NAME`: Name of your project.
+- `YOUR_SERVICE_NAME`: Name of your service.
+- `YOUR_BEARER_TOKEN`: API token for authentication.
 - `follower_fetching={"enabled": true}`: Enables the follower fetching feature.
 
+</TabItem>
+<TabItem value="terraform" label="Terraform">
+
+1. Add the Aiven Terraform provider to the `required_providers` block in your Terraform
+configuration:
+
+   ```hcl
+   terraform {
+    required_providers {
+      aiven = {
+        source  = "aiven/aiven"
+        version = ">=4.0.0, < 5.0.0"
+      }
+    }
+   }
+   ```
+
+1. Set the service connection attributes in the provider block:
+
+   ```hcl
+   provider "aiven" {
+    api_token = "YOUR_API_TOKEN"
+    }
+   ```
+
+   The `api_token` is your Aiven API token for [authentication](/docs/platform/howto/create_authentication_token).
+
+1. Enable follower fetching in your Aiven for Apache Kafka service using the
+following configuration:
+
+   ```hcl
+   resource "aiven_kafka" "example_kafka" {
+    project      = "YOUR_PROJECT_NAME"
+    cloud_name   = "cloud_region"
+    plan         = "business-4"
+    service_name = "example-service-name"
+
+    kafka {
+      # Other Kafka configurations...
+
+      user_config {
+        follower_fetching = {
+          enabled = true
+        }
+      }
+     }
+    }
```

Comment:
I think it should be

```suggestion
   resource "aiven_kafka" "example_kafka" {
    project      = "YOUR_PROJECT_NAME"
    cloud_name   = "cloud_region"
    plan         = "business-4"
    service_name = "example-service-name"

    kafka_user_config {
      # Other Kafka configurations...

      follower_fetching = {
        enabled = true
      }
     }
    }
```

based on https://registry.terraform.io/providers/aiven/aiven/latest/docs/resources/kafk

Reply:
Thanks! Updated the code snippet.